### DISCUSSION TITLE
Is guest mode really safe?

### DISCUSSION THREAD

**Discussion by withgem** [Lakshan-Madushanka/laravel-comments#14] [General]
I noticed that the package uses **IP addresses** to authenticate guest mode users. However, **IP addresses** aren't an accurate way to do this. I won't go into details, but a simple Google search can provide a broad understanding of the pitfalls.

**Comment by Lakshan-Madushanka** [comment]
Please refer below resources,

- [Proposal](https://github.com/Lakshan-Madushanka/laravel-comments/discussions/7)
- https://www.reddit.com/r/PHP/comments/1ehy2hg/uniquely_identify_users_in_guest_mode_without/

I am implementing suggested solution these days.

**Comment by Lakshan-Madushanka** [comment]
Secured in [v2](https://github.com/Lakshan-Madushanka/laravel-comments/releases/tag/2.0.0)
